config/v1/types_cluster_version.go

```diff
@@ -254,6 +254,14 @@ const (
 	// collects anonymized information about the cluster to generate
 	// recommendations for possible cluster issues.
 	ClusterVersionCapabilityInsights ClusterVersionCapability = "Insights"
+
+	// ClusterVersionCapabilityStorage manages the storage operator which
+	// is responsible for providing cluster-wide storage defaults
+	// WARNING: Do not disable this capability when deployed to
+	// bare metal clusters, RHEV and Openstack without reading the docs.
```

Comment:
Please remove note about bare metal clusters - cluster-storage-operator actually does not do anything there. I would even recommend to disable the capability on bare metal.

OpenStack and RHEV is correct.